### PR TITLE
Integration of Checkout, Calendar and Code coverage

### PR DESCRIPTION
**Pull request by IamTechknow**
MVP is reached for my component! Post-integration tasks done: Used React Router, CSS loaders, and focusing on test coverage. README is updated.

### CODE REVIEW

File: test/__tests__/CalendarTest.jsx [hunk 1]
```diff
@@ -61,11 +86,79 @@ describe('Calendar headeronent test suite', function() {
...
+    calendar.setProps({month, year});
     calendar.update();
-    
+
     rows = calendar.find('.checkoutCalRow');
     expect(rows.length).toBe(5); // October 2018 has 5 rows in the calendar
   });
 
+  it('should display dashes on all cells before today', function() {
+    var thisMonth = new Date().getMonth();
```
Comment (keshavmandayam) [inline test/__tests__/CalendarTest.jsx]: You might want to switch to const to get more in line with airbnb but not significant

File: client/components/CheckoutCalendar.jsx [hunk 2]
```diff
@@ -85,22 +143,47 @@ export default class CheckoutCalendar extends React.Component {
...
+  
+  onResetDates() {
+    this.setState({
+      checkinDay: null,
+      checkoutDay: null
+    });
+  }
 
   render() {
+    var checkinStr = this.state.checkinDay ? new Date(this.state.checkinDay).toLocaleDateString() : 'Check in';
```
Comment (keshavmandayam) [inline client/components/CheckoutCalendar.jsx]: May want to go with const again.

**Review by keshavmandayam — CHANGES_REQUESTED** [review]
Put in a couple simple airbnb conformity suggestions but otherwise nothing noticeably off. Very nice!

File: test/__tests__/CalendarTest.jsx (hunk 1 above)
Comment (IamTechknow) [reply test/__tests__/CalendarTest.jsx]: Done, used let and also const when possible

File: client/components/CheckoutCalendar.jsx (hunk 2 above)
Comment (IamTechknow) [reply client/components/CheckoutCalendar.jsx]: Done!

**Pull request merged**